packages/api-cardano-db-hasura/hasura/project/migrations/1589369664961_init/up.sql

```diff
@@ -59,13 +59,13 @@ FROM epoch;
 CREATE VIEW "Reward" AS
 SELECT
   reward.amount AS "amount",
-  reward.id AS "id",
   (
 	  SELECT stake_address.view
 	  FROM stake_address
 	  WHERE stake_address.id = reward.addr_id
   ) AS "address",
-  reward.tx_id AS "tx_id"
+  reward.epoch_no AS "epochNo",
+  ( SELECT pool_hash.hash FROM pool_hash WHERE pool_hash.id = reward.pool_id ) AS "pool_hash"
```

Comment:
Minor and won't block review, but we should stick to one format for column headers - either camalCase or under_score

Reply:
The format is to align with the desired GraphQL model since Hasura derives field names from the column names. I elected to not use camelCase for fields that are only included in the view for establishing relationships, and therefore not exposed over GraphQL.